hugolib/multilingual.go

```diff
@@ -0,0 +1,48 @@
+package hugolib
+
+import (
+	"github.com/spf13/cast"
+	"github.com/spf13/viper"
+)
+
+type Multilingual struct {
+	enabled bool
+	config  *viper.Viper
+
+	Languages []string
+}
+
+func (ml *Multilingual) GetString(key string) string { return cast.ToString(ml.Get(key)) }
+func (ml *Multilingual) GetStringMap(key string) map[string]interface{} {
+	return cast.ToStringMap(ml.Get(key))
+}
+
+func (ml *Multilingual) GetStringMapString(key string) map[string]string {
+	return cast.ToStringMapString(ml.Get(key))
+}
+
+func (ml *Multilingual) Get(key string) interface{} {
```

Comment:
We have to start somewhere.  I'm not pretending the thing is perfect.  But i'd like to let the community give it a try.

This would not break existing sites.. only potentially take more time when community members start building multilingual sites, as we sort out all the kinks. I know it's a big chunk.. but aren't we better to bring it in sooner in the release process, rather than close to a release date?

If I list in the documentation all the keys that are picked-up under `Multilingual`, would that be satisfying ?  I could try to list the keys that are explicitly _not_ taken into account.. if I can build such a list.  Otherwise, I'm a bit blocked.. as I'm not sure what else would need to be done.